packages/manager/modules/overthebox/src/overthebox/remote/overTheBox-remote.routing.js

```diff
@@ -0,0 +1,12 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('overTheBoxes.overTheBox.remote', {
+    url: '/remote',
+    views: {
+      otbView: 'overTheBoxRemote',
+    },
+    resolve: {
+      serviceName: /* @ngInject */ ($transition$) =>
```

Comment:
This could be declared in the parent state to avoid to duplicate the declaration, it will be easier to maintain if the param name change